### ISSUE TITLE
ZeroDivisionError

### ISSUE THREAD

**Issue by Hongchenglong** [hanglearning/VBFL#15]
If TP equals zero, the following error will occur:

```
worker device_5 has received total 0 rewards for this comm round.
There are no available local params for device_5 to perform global updates in this comm round.
Traceback (most recent call last):
  File "/content/drive/My Drive/VBFL/main.py", line 809, in <module>
    processing_time = device.process_block(device.return_the_added_block(), log_files_folder_path, conn, conn_cursor)
  File "/content/drive/My Drive/VBFL/Device.py", line 634, in process_block
    precision = TP/(TP + FP)
ZeroDivisionError: division by zero
```

**Comment by Hongchenglong** [comment]
This is my fix #16.

**Comment by hanglearning** [comment]
Thanks! This is a feature I later added for presentation. Should've thought about the edge case!
